stock_helper_delivery/models/stock_picking.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2021 Camptocamp SA (http://www.camptocamp.com)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from odoo import fields, models
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    def get_carrier_from_chained_picking(self):
```

Comment:
as reported on the other PR: make it return the picking instead. To be renamed as `get_picking_with_carrier_from_chain` or something like that.